packages/js/src/settings/routes/site-representation.js

```diff
@@ -241,14 +247,16 @@ const SiteRepresentation = () => {
 															// eslint-disable-next-line react/jsx-no-bind
 															onClick={ arrayHelpers.remove.bind( null, index ) }
 															className="yst-mt-7 yst-p-2.5"
+															// translators: %1$s expands to array index + 1.
+															aria-label={ sprintf( __( "Remove Other profile %1$s", "wordpress-seo" ), index + 1 ) }
```

Comment:
Do we want `Other` with capital here? 🤔 Does that make sense as a name of the field? I don't know!